spec/statement.dd

```diff
@@ -1798,7 +1798,7 @@ $(GNAME AsmInstructionList):
 )
 
 $(P An asm statement enables the direct use of assembly language instructions.
-This makes it easy to obtain direct access to special CPU features without
+This makes it easy to obtain direct access to special [CPU](https://hosting.review/web-hosting-glossary/#19) features without
```

Comment:
```suggestion
This makes it easy to obtain direct access to special [CPU](https://en.wikipedia.org/wiki/Central_processing_unit) features without
```

Why not link to Wikipedia instead?

Reply:
I think this link https://hosting.review/web-hosting-glossary/#19 explains the term "CPU" in a language of beginners. I can say from my own perspective that it's better to explain it in easy words. That means they can understand this term better since that link also displays the useful illustrations.

Reply:
Easy words may be useful to explain for beginners.
But easy language does not change the fact that this statement is just **wrong**.